WINDOWS.md

```diff
@@ -512,12 +512,19 @@ For each of these extensions:
 >\- In the browser, accept to use VS Code to install the extension
 >\- In VS Code, click on `install`
 
+For everyone
+- [Sublime Text Keymap](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings)
 
+For the Web Development bootcamp:
 - [Rails Snippets](https://marketplace.visualstudio.com/items?itemName=hridoy.rails-snippets)
 - [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 - [ERB Helper Tags](https://marketplace.visualstudio.com/items?itemName=rayhanw.erb-helpers)
 - [ruby-rubocop](https://marketplace.visualstudio.com/items?itemName=misogi.ruby-rubocop)
+
+For the Data Science bootcamp:
 - [Python](https://marketplace.visualstudio.com/items?itemName=ms-python.python)
+- [Jupyter](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
+- [Python Indent](https://marketplace.visualstudio.com/items?itemName=KevinRose.vsc-python-indent)
```

Comment:
@gmanchon : we don't use this repo for data student setup right? We only use https://github.com/lewagon/data-setup ? If so, let's separate concern and avoid mentioning `for everyone`, `web development` or `data science` here.

Reply:
@dmilon these modifications where included when I ran `ruby build.rb`, so I assume that they were included by a previous commit...

The partials of this (web) setup repo are used by the data-setup repo as well. The partials concerned are the one listed in the build.rd file and having a name starting by "setup/" : https://github.com/lewagon/data-setup/blob/master/build.rb

Reply:
:ok_hand: